src/routers/productExports.ts

```diff
@@ -0,0 +1,22 @@
+import * as express from 'express';
+
+import createRouter from '@/utilities/functions/createRouter';
+import authorized from '@/middleware/authorized';
+import { UserType } from '@/models/User/UserType';
+import { StatusCode } from '@/models/statusCodes';
+
+const router = createRouter();
+
+router.get('/', async (req, res) => {
+  res.status(StatusCode.OK).send('Successfully retrieved all product exports') ;
+}, authorized(UserType.ADMIN));
+
+router.post('/', async (req, res) => {
+  res.status(StatusCode.CREATED).send('Successfully retrieved new product export');
```

Comment:
This and line 19 don't need the word retrieved in it I think. So it should be just "successfully created new product export" so that it is clear that it isn't a GET. (Retrieved implies a GET imo)

Reply:
Copy paste artifacts :)